src/kademlia/node.cpp

```diff
@@ -1152,4 +1157,14 @@ void node::incoming_request(msg const& m, entry& e)
 		return;
 	}
 }
+
+void node::update_counters()
+{
+	const dht_storage_counters& dht_cnt = m_storage->counters();
+	m_counters.set_value(counters::dht_torrents, dht_cnt.torrents);
```

Comment:
or actually, if we would return the dht_storage_counters object here, the DHT could potentially be free from the counters dependency (one step closer to being able to be free-standing)